cli/src/wallet.rs

```diff
@@ -524,15 +527,29 @@ fn process_airdrop(
         .retry_get_balance(&config.keypair.pubkey(), 5)?
         .unwrap_or(previous_balance);
 
-    Ok(format!("Your balance is: {:?}", current_balance))
+    let ess = if current_balance == 1 { "" } else { "s" };
+    Ok(format!(
+        "Your balance is: {:?} lamport{}",
+        current_balance, ess
+    ))
 }
 
-fn process_balance(pubkey: &Pubkey, rpc_client: &RpcClient) -> ProcessResult {
+fn process_balance(
+    pubkey: &Pubkey,
+    rpc_client: &RpcClient,
+    use_lamports_unit: &bool,
+) -> ProcessResult {
     let balance = rpc_client.retry_get_balance(pubkey, 5)?;
     match balance {
         Some(lamports) => {
-            let ess = if lamports == 1 { "" } else { "s" };
-            Ok(format!("{:?} lamport{}", lamports, ess))
+            let msg = if *use_lamports_unit {
```

Comment:
This block could be turned into a helper function and reused in process_airdrop and process_show_account